### PR TITLE
Dukto: add recipe

### PR DESCRIPTION
**Pull request by X547**

### CODE REVIEW

File: net-misc/dukto/dukto-6.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+CHECKSUM_SHA256="599e808fb9206f24386afb32ee96766797d4354b5f34838d8b4dae65f446fe5a"
+SOURCE_DIR="dukto-00301800b3a645b80361a59de911839bf49ee32e"
+PATCHES="dukto-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	dukto$secondaryArchSuffix = $portVersion
+	app:Dukto$secondaryArchSuffix = $portVersion
```
Comment (Begasus) [inline net-misc/dukto/dukto-6.0.0.recipe:21]: Haven't checked the build (yet), but secondaryArchSuffix shouldn't be needed here:
```diff
- 	app:Dukto$secondaryArchSuffix = $portVersion
+ 	app:Dukto
```

File: net-misc/dukto/dukto-6.0.0.recipe [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	make $jobArgs
+}
+
+INSTALL()
+{
+	mkdir $appsDir
+	cp dukto $appsDir/Dukto
+
+	mkdir -p $dataDir/deskbar/menu/Applications
+	addAppDeskbarSymlink "$appsDir/Dukto" "Dukto"
```
Comment (diversys) [inline net-misc/dukto/dukto-6.0.0.recipe:66]: Can be replaced with `addAppDeskbarSymlink $appsDir/Dukto`.

**Comment by Begasus** [comment]
Builds ok, launches ok, other then that I have no clue how to use it ;)

**Comment by X547** [comment]
> other then that I have no clue how to use it ;)

Open second instance of Dukto on second PC or virtual machine connected by LAN and you will see new item in destination list. Now you can select destination and send text/files. Files can be send by drag&drop. By default files are received to home folder.

**Pull request merged**